resources/seqr_metadata_templates/README.md

```diff
@@ -0,0 +1,275 @@
+# Organising rare disease data and metadata for seqr <!-- omit from toc -->
+
+![CPG Logo](../images/cpg_logo_1280x329.png)
+
+## Table of Contents <!-- omit from toc -->
+
+---
+
+- [1. Genomic Data](#1-genomic-data)
+- [2. Pedigree\_template](#2-pedigree_template)
+- [3. Families\_metadata\_template (Optional)](#3-families_metadata_template-optional)
+- [4. Individuals\_metadata\_template](#4-individuals_metadata_template)
+- [5. Sample\_mapping\_template](#5-sample_mapping_template)
+
+---
+
+### i. Purpose
+
+The purpose of this document is to provide instructions on how to prepare project specific data and metadata for use in seqr<br /> related to an individual's family history (pedigree).
+
+<br />
+
+### ii. Background
+
+The CPG utilises four distinct metadata files to provide information about samples to the variant curation team as they <br /> perform variant analysis in seqr.
```

Comment:
Do we need the inline `<br />`s? It will handle wrapping natively wont it?

Reply:
Yeah, I think I got a bit fixated on making everything appear nice in the markdown preview while I was writing this up that I completely forgot about all the different devices and screen sizes this will be viewed from. 
I'll get rid of some of these since they will look silly with the native wrapping in there as well